src/datadog/azext_datadog/generated/_help.py

```diff
@@ -0,0 +1,343 @@
+    examples:
+      - name: LinkedResources_List
+        text: |-
+               az datadog linked-resource list --monitor-name "myMonitor" --resource-group "myResourceGroup"
+"""
+
+helps['datadog monitored-resource'] = """
+    type: group
+    short-summary: datadog monitored-resource
+"""
+
+helps['datadog monitored-resource list'] = """
+    type: command
+    short-summary: "List the resources currently being monitored by the Datadog monitor resource."
+    examples:
+      - name: MonitoredResources_List
+        text: |-
+               az datadog monitored-resource list --monitor-name "myMonitor" --resource-group "myResourceGroup"
+"""
+
+helps['datadog monitor'] = """
+    type: group
+    short-summary: datadog monitor
+"""
+
+helps['datadog monitor list'] = """
+    type: command
+    short-summary: "List all monitors under the specified subscription."
+    examples:
+      - name: Monitors_ListByResourceGroup
+        text: |-
+               az datadog monitor list --resource-group "myResourceGroup"
+"""
+
+helps['datadog monitor show'] = """
+    type: command
+    short-summary: "Get the properties of a specific monitor resource."
+    examples:
+      - name: Monitors_Get
+        text: |-
+               az datadog monitor show --name "myMonitor" --resource-group "myResourceGroup"
+"""
+
+helps['datadog monitor create'] = """
+    type: command
+    short-summary: "Create a monitor resource."
+    parameters:
+      - name: --datadog-organization-properties
+        long-summary: |
+            Usage: --datadog-organization-properties linking-auth-code=XX linking-client-id=XX enterprise-app-id=XX
+
+            linking-auth-code: The auth code used to linking to an existing datadog organization.
+            linking-client-id: The client_id from an existing in exchange for an auth token to link organization.
+            enterprise-app-id: The Id of the Enterprise App used for Single sign on.
+      - name: --user-info
+        long-summary: |
+            Usage: --user-info name=XX email-address=XX phone-number=XX
+
+            name: Name of the user
+            email-address: Email of the user used by Datadog for contacting them if needed
+            phone-number: Phone number of the user used by Datadog for contacting them if needed
+    examples:
+      - name: Monitors_Create
+        text: |-
+               az datadog monitor create --name "myMonitor" --sku-name "myMonitor" --location "West US 2" \
+--datadog-organization-properties name="myOrg" enterprise-app-id="00000000-0000-0000-0000-000000000000" \
+linking-auth-code="someAuthCode" linking-client-id="00000000-0000-0000-0000-000000000000" subscription="pro" \
+--monitoring-status "Enabled" --user-info name="Alice" email-address="alice@microsoft.com" phone-number="123-456-7890" \
+--sku-name "drawdown_testing_20200904_Monthly" --tags Environment="Dev" --resource-group "myResourceGroup"
```

Comment:
`--sku-name` appear twice in this example with different values.

Reply:
Updated.